cluster/addons/rbac/node-bootstrap-binding.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: node-bootstrap
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:node-bootstrapper
+subjects:
+- apiVersion: rbac/v1beta1
+  kind: Group
+  name: system:node-bootstrapper
+- apiVersion: rbac/v1beta1
+  kind: Group
+  name: system:node
```

Comment:
@jcbsmpsn is working on implementing rotation that will use the kubelet's certificate as credentials. I can omit this until it is impelmented.